core/src/main/java/hudson/DNSMultiCast.java

```diff
@@ -66,7 +66,8 @@ public Object call() {
                     jmdns.registerService(ServiceInfo.create("_http._tcp.local.","Jenkins",
                             jenkins_port,0,0,props));
                 } catch (IOException e) {
-                    LOGGER.log(Level.WARNING,"Failed to advertise the service to DNS multi-cast",e);
+                    LOGGER.log(Level.INFO, "Cannot advertise service to DNS multi-cast, skipping: {0}", e);
```

Comment:
This looks incorrect as it will use [`Logger.log(java.util.logging.Level, java.lang.String, java.lang.Throwable`)](https://docs.oracle.com/javase/7/docs/api/java/util/logging/Logger.html#log(java.util.logging.Level,%20java.lang.String,%20java.lang.Throwable)) and not the message formatter that you are expecting - causing the next line to be a pointless reprint of the same exception at a different logging level.